### PR TITLE
popup-window

### PR DESCRIPTION
**Pull request by asadmisrani**
Scripts.js file and README.md file were modified

Use basic JavaScript syntax.
Use JavaScript to manipulate DOM elements.
Use JavaScript events.
Understand how to parse a Figma design to create a UI.
Use objects to store and access data.

### CODE REVIEW

**Review by Zuhaib042 — COMMENTED** [review]
Hi @asadmisrani  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have a commit from your partner. if he doesn't want to work with you then you can contact student success team by clicking on the support button on the dashboard.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by nedjwab — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @asadmisrani 

Nice job, you are almost there.



## (Highlights) Good Points: :
- Good implementation of some of the project requirements  ✔
- GitHub flow was correctly used ✔.

## (Changes Required) Aspects to improve: ♻️
You did great implementing some of the project requirements, but you still have some issues: 
[ ] kindly fix the image in your pop up window , it's not showing up :

 Your design 👇 : 
his


` note:check the URL in the javascript array`

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by asadmisrani** [comment]
The image in the pop-up window was fixed, Now it's showing up.

File: script.js [hunk 1]
```diff
@@ -1,7 +1,19 @@
...
+    image: './images/screenshots/MainPage.png',
+    image1: './images/screenshots/BooksList.png',
+    description1: 'An application that allows users to add or remove books from a list, not subscription is needed nor online conection.',
+    description: 'Awesome Books is a website that allows users to add/remove books from a list. It was achieved by using JavaScript objects and arrays. Dynamically modify the DOM and added basic events, using ES6 sintax, using modules into javascript code, and use of arrow functions.',
+    tech: ['html', 'css', 'javaScript', 'Bootstrap', 'ES6 syntax'],
+    seeLive: 'https://asadmisrani.github.io/AwesomeBooks/',
+    seeSource: 'https://github.com/asadmisrani/AwesomeBooks-ES6',
+    class: 'popCard1',
+  },
   {
```
Comment (Shedrack-Sunday) [inline script.js:17]: - [ ] I can see you did make changes asked by the previous reviewer, however, the pictures on the pop-up after the first card are not still displaying. Please check the file paths

## View:
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @asadmisrani     👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing the requirements,  However, there are a few you need to work on, to move to the next project. 💯

 


## Highlights
- Proper use CSS ✔️ 
- Nice README structure 👍 
- Moderate design 💯 
  
# Required Changes ♻️

- There's a bit content structure that defers i reference to the figma design given which the previous reviewer pointed 

_check the comment section for a detailed review_ 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by asadmisrani** [comment]
Are you not seeing this picture that is working fine and perfectly[image:
Screenshot_1.png]
?

File: script.js (hunk 1 above)
Comment (Nemwel-Boniface) [inline script.js:17]: - [ ] I went further to investigate and realized in your object for the first card has the field for ```image1``` which is not present in all other objects. When I duplicated the ```image1``` in the second and third objects, the popup displayed the images as it is supposed to. 

[Screencast from 13-04-2023 07:25:08 ASUBUHI.webm](https://user-images.githubusercontent.com/86318284/231652893-0c997b1b-d817-4ac6-b4a9-ea67a869c3ac.webm)

**Review by Nemwel-Boniface — COMMENTED** [review]
## Hello @asadmisrani   👋,

I am Nemwel your reviewer.


## _This is an additional review as a result of the second opinion request._

I have received your second opinion request and inspected your project based on the complaint of the [last reviewer's](https://github.com/asadmisrani/portfoliofinishmobileversion.github.io/pull/14#discussion_r1164825550) (@Shedrack-Sunday ) request. 

**From my findings,** the changes requested by him are actually valid and indeed the images do not show on the popup. Below I have attached a screencast for you.

[Screencast from 13-04-2023 07:18:01 ASUBUHI.webm](https://user-images.githubusercontent.com/86318284/231652543-3066cd8b-24a9-4df1-b8fd-59be04689023.webm)

**Therefore** as the changes requested are actually valid, kindly ensure that you fix the images not displaying on all the cards by adding the ```image1``` field on all the objects.

Kindly make the required changes and when you are ready, please request another review in your dashboard.

In case anything is not 100% clear tag me @Nemwel-Boniface so that I get the notification.

Keep rocking 🚀!

**Comment by asadmisrani** [comment]
Done.
I have duplicated the first card has a field for Image1 which is now
present in all other objects. Now the popup displayed the images as it is
supposed to.

File: script.js [hunk 2]
```diff
@@ -1,7 +1,19 @@
 // PopUp Object
 
 const popObjects = [
-
+  {
+    title: 'Awesome books',
+    subtitle: 'Microverse',
```
Comment (joskalenda) [inline script.js:6]: - [ ] @asadmisrani, I think the same issue is still persisting, you should see how to fix it on the go, see the record below and make the necessary changes to it


- Seeing this record, the image only displays on the 1st card, the rest does not show up.

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @asadmisrani , |
|Great work on making the changes requested by a previous reviewer 👏, You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet. |
|  |

## Highlights

- [ ].. ✅ Good design
- [ ].. ✅ Green linters
- [ ].. ✅ Good commit messges

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by asadmisrani** [comment]
All images are working fine I have checked.
what are you checking? what happened? to you I lost my 3 reviews and I
don't want to use extra reviews.
This is my live link check this everything is working fine:
https://asadmisrani.github.io/portfoliofinishmobileversion.github.io/
I don't know why are you doing this I am stuck with you.

**Review by Nemwel-Boniface — COMMENTED** [review]
## Hello @asadmisrani   👋,

I am Nemwel your reviewer.


## _This is an additional review as a result of the second opinion request._

**Note:** When you request for a second opinion request, you are required to be patient and not close/ merge a PR or request for another review in your dashboard, or make any new commits until the second opinion request has been resolved unless it will alter the outcome of the second opinion request.

I have received your second opinion request and inspected it and from my end the popup does show the images as it is supposed to. For some reason the issue might be arising from the code reviewer's end.

**Note** However, I believe that there is room for improvement and you could consider giving the popup a height of 100vh so that is fits on the whole page without you having to scroll all the way up to view the popup. Video shown below:

[Screencast from 13-04-2023 03:08:37 ALASIRI.webm](https://user-images.githubusercontent.com/86318284/231753852-7a65016f-4b93-4837-abac-92901f9cee26.webm)


As this was the only reason that your project was not approved it will be approved as soon as possible.
Please note that since we will approve it manually so this could take a few hours!

In case anything is not 100% clear tag me @Nemwel-Boniface so that I get the notification.

Keep rocking 🚀!

**Comment by franclobo** [comment]
Hi @asadmisrani, 

# STATUS: APPROVED 😃 

git-merge

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- No linter errors ✔️ 
- Descriptive PR ✔️ 
- Correct Github flow ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**